library/CLAUDE.md

```
@@ -0,0 +1,120 @@
+
+## How to Build & Test
+
+```bash
+./gradlew :library:test              # Run all tests (JUnit 5 + Robolectric)
+./gradlew :library:lint              # Lint (warnings = errors)
+./gradlew :library:assembleDebug     # Build AAR
+```
+
+## Data Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)